NineChronicles.Headless.Executable/Program.cs

```diff
@@ -346,7 +350,20 @@ static void ConfigureSentryOptions(SentryOptions o)
         private Guid? LoadAWSSinkGuid()
         {
             string path = AWSSinkGuidPath();
-            return File.Exists(path) ? Guid.Parse(File.ReadAllText(AWSSinkGuidPath())) : (Guid?)null;
+            if (!File.Exists(path))
+            {
+                Console.Error.WriteLine($"AWSSink id doesn't exist.");
```

Comment:
I guess displaying `path` is useful.

```suggestion
                Console.Error.WriteLine($"AWSSink id doesn't exist. (path: {path})");
```